File: beginner/1013.ts
import * as fs from 'fs';
import * as path from 'path';

const myPath = path.join(__dirname, 'file.txt');
const input = fs.readFileSync(myPath, 'utf8');

const valoresInputString = input.split(' ');
const valoresInput = valoresInputString.map(valor => Number(valor));

const maiorAB = (valoresInput[0] + valoresInput[1] + Math.abs(valoresInput[0] - valoresInput[1])) / 2;

const verificaMaior = (valor1: number, valor2: number) => {
  return valor1 > valor2 ? valor1 : valor2;
};

console.log(verificaMaior(maiorAB, valoresInput[2]) + ' eh o maior');

File: beginner/1013.c
#include <stdio.h>
#include <stdlib.h>

int main() {
  int a, b, s;
  scanf("%d%d%d", &a, &b, &s);
  int major = (a + b + abs(a - b)) / 2;
  if (major < s) {
    major = s;
  }
  printf("%d eh o maior\n", major);
  return 0;
}
